Application/EdFi.Ods.SandboxAdmin/Startup.cs

```diff
@@ -191,10 +192,19 @@ public void ConfigureContainer(ContainerBuilder builder)
         }
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-        public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILoggerFactory loggerFactory)
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILoggerFactory loggerFactory, IConfigurationRoot configuration)
         {
             loggerFactory.AddLog4Net();
 
+            var pathBase = configuration["PathBase"];
+
+            if (!string.IsNullOrEmpty(pathBase))
+            {
+                pathBase = pathBase.Replace("/", "");
```

Comment:
Doing the slash handling this way is dangerous if they want to use a multipart path base such as "api/test".  This should only be normalizing trailing and leading slashes.